tests/Unit/IO/H5/Python/Test_ExtractDatFromH5.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
```

Comment:
I wouldn't bother keeping the expected files in the repo, only the h5 file. You can check that the correct directory structure was written and use `np.load_txt` to check the correct data was written. It should be relatively easy to also just read in the legend. This will fix the long line problem too, since you don't need 16 digits in the numpy arrays you want to compare to :)